docs/tutorials/credentials-revocation/README.md

```diff
@@ -43,7 +43,7 @@ Create a new `POST` request called "Revoke Credentials" in the "Credentials Revo
   ```json
   {
     "credentialId": "{{verifiable_credential_id}}",
-    "credentialStatus": [{ "type": "RevocationList2020Status", "status": "1" }]
+    "credentialStatus": [{ "type": "StatusList2021Entry", "statusPurpose": "revocation", "status": "1" }]
```

Comment:
this could have been a "new suite"... that probably would have been a smarter way to add support.

Reply:
Thanks, made those changes in [this](https://github.com/w3c-ccg/traceability-interop/pull/559) PR